tests/snmpsim/ceraos.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.2.0|6|1.3.6.1.4.1.2281.1.20.2.2
```

Comment:
This sysDescr does not apply to all CeraOS devices, it only applies to one subfamily, the IP-20C.
sysDescr varies or is not present on other varients. Will this affect discovery etc?